translations/messages+intl-icu.fr_FR.yaml

```diff
@@ -425,4 +427,4 @@ users.no_organization_authorization: 'Cet utilisateur ne peut pas créer de tick
 users.no_organization: 'Aucune organisation'
 users.organization: Organisation
 users.password: 'Mot de passe'
-users.yourself: vous
+users.yourself: vous
```

Comment:
You removed the new line character at the end of this file. You must not. It is very often recommended (if not required) to keep it so when you open several files in the terminal (with `cat` for instance), the lines of each file are correctly displayed. You can learn more about it on StackOverflow: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline